ecaudit/src/test/java/com/ericsson/bss/cassandra/ecaudit/metrics/TestAuditMetrics.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright 2019 Telefonaktiebolaget LM Ericsson
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.ericsson.bss.cassandra.ecaudit.metrics;
+
+import java.util.concurrent.TimeUnit;
+import java.util.function.Function;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.codahale.metrics.Timer;
+import org.apache.cassandra.metrics.CassandraMetricsRegistry;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.StrictStubs.class)
+public class TestAuditMetrics
+{
+    private static final String GROUP_NAME = "com.ericsson.bss.cassandra.ecaudit";
+    private static final String METRIC_TYPE = "Audit";
+    private static final String METRIC_SCOPE = "ClientRequests";
+    private static final String METRIC_NAME_FILTER = "AuditFilter";
+    private static final String METRIC_NAME_AUDIT = "Audit";
+
+    @Mock
+    private Function<CassandraMetricsRegistry.MetricName, Timer> mockTimerFunction;
+
+    @Before
+    public void init()
+    {
+        when(mockTimerFunction.apply(any())).thenReturn(mock(Timer.class));
+    }
+
+    @Test
+    public void testFilterRequestTiming()
+    {
+        Timer mockTimer = mock(Timer.class);
+        CassandraMetricsRegistry.MetricName metric = createMetricName(METRIC_TYPE, METRIC_NAME_FILTER, METRIC_SCOPE);
+
+        when(mockTimerFunction.apply(eq(metric))).thenReturn(mockTimer);
+
+        AuditMetrics auditMetrics = new AuditMetrics(mockTimerFunction);
+        verify(mockTimerFunction, times(1)).apply(eq(metric));
```

Comment:
use the verify-method without the VerificationMode-argument? Since times(1) it the default value

Reply:
Done